### ISSUE TITLE
Won't send bitcoin, blockchain sync is stuck

### ISSUE THREAD

**Issue by kev-vl** [hodlwallet/hodl-wallet-ios#68]
I have an issue where transactions aren't going through. The rescanning process proposed by the app does not make any progress, after a while the app crashes. What do you recommend I do?

Also, when recovering the wallet in another app, what derivation path should I go with?

**Comment by igorgue** [comment]
Hi @kev-vl,

I apologize for the error, I will check the crash report but there's no guarantee I'll get to fix it since these errors are very rare and the logs are very confusing but they do happen from user to user. The advantage of HODL using the P2P network is a problem sometimes, thus wallets like Electrum or many others sacrifice your privacy with a server model.

The rescanning process will only progress if the app is in open and left there without the screen going to sleep, I usually let it do it by itself with my phone connected overnight, sometimes it may take a while.

Our derivation path is `m/0'` we have had success using Electrum to recover funds, you must create a wallet for legacy addresses (same seed same path) and for Segwit / bech32 addresses (same seed and path).

**Comment by kev-vl** [comment]
Thank Igor, I resolved the issue by recovering my wallet in another device's HODL wallet, which worked just fine. On my phone (iOS), the rescanning process did not progress even with the app open, active screen. I think the app itself prevents the screen from going to sleep, so that's good. 

I found out that the main problem was a many-input tx that wouldn't go through. Breaking it down in multiple smaller tx's allowed me to move my coins. Perhaps the issue was an insufficient tx-fee for the amount of bytes, putting it below the 1 sat/(v)byte threshold, idk. The app then suggested a blockchain rescan, and that's where the trouble started for me. 

It could be helpful to have more elaborate error-messages or progress indicators, because the user is kind of left in the dark on why some operations fail.
